rollup/sync_service.go

```diff
@@ -859,13 +905,20 @@ func (s *SyncService) ProcessTransactionEnqueuedLog(ctx context.Context, ethlog
 	// Nonce is set by god key at execution time
 	// Value and gasPrice are set to 0
 	tx := types.NewTransaction(uint64(0), event.Target, big.NewInt(0), event.GasLimit.Uint64(), big.NewInt(0), event.Data, &event.L1TxOrigin, new(big.Int).SetUint64(ethlog.BlockNumber), types.QueueOriginL1ToL2, types.SighashEIP155)
-	// Timestamp is used to update the blockchains clocktime
-	timestamp := time.Unix(event.Timestamp.Int64(), 0)
-	rtx := RollupTransaction{tx: tx, timestamp: timestamp, blockHeight: ethlog.BlockNumber, executed: false, index: event.QueueIndex.Uint64()}
+	// Set the timestamp in the txmeta
+	timestamp := uint64(event.Timestamp.Int64())
+	tx.SetL1Timestamp(timestamp)
```

Comment:
What is the difference between `SetL1Timestamp` and `SetLatestL1Timestamp` below?

Reply:
`SetLatestL1Timestamp` is a method on the `SyncService`, it handles the global value that the syncservice tracks, while `SetL1Timestamp` is a method on the transaction struct itself to set the field on the tx